metadata-ingestion/source_docs/bigquery.md

```diff
@@ -1,14 +1,17 @@
 # BigQuery
 
+To get all metadata from BigQuery you need to use two plugins `bigquery` and `bigquery-usage`. Both of them are described in this page. These will require 2 separate recipes. We understand this is not ideal and we plan to fix this problem in future to have a single plugin.
+
 For context on getting started with ingestion, check out our [metadata ingestion guide](../README.md).
 
-## Setup
+## `bigquery` plugin
```

Comment:
nit: just say `bigquery`, drop the "plugin" suffix

Reply:
(it's already known to be a plugin due to the page context)

Reply:
changed